download_checkpoint.py

```
@@ -0,0 +1,40 @@
+import argparse
+import os
+
+from huggingface_hub import HfFolder, snapshot_download
+
+# from huggingface_hub import HfApi
+
+
+def main(args):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)